### PR TITLE
Link updates

### PR DESCRIPTION
**Pull request by ManasiGandhi**

### CODE REVIEW

**Review by Rwalls1 — APPROVED** [review]

File: modules/ROOT/pages/external-configuration.adoc [hunk 1]
```diff
@@ -40,15 +40,15 @@ Thus, it is important to change the configuration of microservices without updat
 
 == MicroProfile Config
 
-xref:reference:feature/mpConfig-1.4.adoc[MicroProfile Config] separates the configuration from code.
+xref:mpConfig-1.4.adoc[MicroProfile Config] separates the configuration from code.
```
Comment (dmuelle) [inline modules/ROOT/pages/external-configuration.adoc:43]: use the macro so the doc will always resolve to the most recent feature version
`feature:mpConfig[display=MicroProfile  Config]`

File: modules/ROOT/pages/external-configuration.adoc [hunk 2]
```diff
@@ -40,15 +40,15 @@ Thus, it is important to change the configuration of microservices without updat
...
 With MicroProfile Config, you can inject the external configuration into services in the containers without repackaging them.
 
-Applications can use MicroProfile Config as a single API to retrieve configuration information from different sources such as link:reference:config/server-configuration-overview.adoc[system properties, system environment variables], properties files, XML files, or data sources.
+Applications can use MicroProfile Config as a single API to retrieve configuration information from different sources such as link:/docs/21.0.0.2/reference/config/server-configuration-overview.html[system properties, system environment variables], properties files, XML files, or data sources.
```
Comment (dmuelle) [inline modules/ROOT/pages/external-configuration.adoc:46]: to link to the server config overview, use a relative link:

`xref:reference:config/server-configuration-overview.adoc`

File: modules/ROOT/pages/external-configuration.adoc [hunk 3]
```diff
@@ -40,15 +40,15 @@ Thus, it is important to change the configuration of microservices without updat
...
+The same configuration property can be defined in link:/guides/microprofile-config-intro.html#configuring-with-the-properties-file[multiple config sources].
 
-With MicroProfile Config, you can https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#_simple_dependency_injection_example[inject an individual property into a class], or https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#_aggregate_related_properties_into_a_cdi_bean[inject multiple properties in one CDI bean].
-You can also retrieve values from configuration https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#_simple_programmatic_example[without using a CDI bean].
+With MicroProfile Config, you can link:/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#_simple_dependency_injection_example[inject an individual property into a class], or link:/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#_aggregate_related_properties_into_a_cdi_bean[inject multiple properties in one CDI bean].
```
Comment (dmuelle) [inline modules/ROOT/pages/external-configuration.adoc:50]: to link to an external site, you don't need `link:` but you do need `https://download.eclipse.org/` or whatever the URL for the external site is. I think the way you had these should be correct if those are working URLs

**Review by dmuelle — CHANGES_REQUESTED** [review]
changes to link syntrax

**Pull request merged**